### PR TITLE
Forward calorimetry

### PR DESCRIPTION
**Pull request by zaborowska**
Changes proposed in this pull-request:

- forward calorimetry description using the same factory as endcaps
- change of the inner radius of the forward calorimetry in FCChh dimensions so that the calorimeter starts/ends at eta=6 and the inner cryostat has constant thickness

### CODE REVIEW

**Comment by hegner** [comment]
retest this please

File: Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    <!-- hadronic calorimeter: HFWD -->
+    <constant name="HFWD_z1" value="EMFWD_z2 + BathThicknessMiddle"/>
+    <constant name="HFWD_z2" value="CryoFwd_z2 - CryoThicknessBack - BathThicknessBack"/>
+    <constant name="HFWD_rmin1" value="HFWD_z1*tan(2*atan(exp(-6))) - 2*cm + CryoThicknessInner"/>
+    <constant name="HFWD_rmin2" value="CryoFwd_back_rmin1 + CryoThicknessInner - BathThicknessBack*tan(2*atan(exp(-6)))"/>
+    <constant name="HFWD_rmax" value="FwdCal_rmax - CryoThicknessOuter - BathThicknessOuter"/>
+    <constant name="HFWD_lAr_thickness" value="0.1*mm"/>
+    <constant name="HFWD_Cu_thickness" value="4.*cm"/>
+    <constant name="HFWD_readout_thickness" value="1.2*mm"/>
+  </define>
```
Comment (faltovaj) [inline Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml]: Could you please to change to the naming convention we are using for calorimeters? Not to introduce more and more shortcuts :) EMFWD -> EFCAL, HFWD -> HFCAL. Thanks!

**Comment by zaborowska** [comment]
Hi Jana,

thanks for comments.
1) The active volume starts at eta=6 as it was. The cryostat in Werner's file was between eta 6.5 and eta 6, so I believe it was technically unfeasable to make it along those lines. I changed it so it can have a constant thickness. I didn't comment on that there as it is more of a technical change, there is nothing there anymore beside the beampipe.
2) I was thinking of opening a separate PR with sampling fraction for all barrel, endcap and forward.
I can add it here as well, no problem.

Anna

File: Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml (hunk 1 above)
Comment (zaborowska) [inline Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml]: Sure. However how exactly did we choose EFCAL and HFCAL? It actually makes it really confusing to change it in code if each part follows a different schema. Endcap -> Fwd was much easier ;)

**Pull request merged**